robolectric-resources/src/main/java/org/robolectric/res/ResourceTableFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.robolectric.res;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+public class ResourceTableFactory {
+
+  public static ResourceTable newResourceTable(String packageName, ResourcePath... resourcePaths) {
```

Comment:
Still `static` so there's no way to override behavior, but let's address it later if we decide it's important.